.github/workflows/ci.yml

```
@@ -0,0 +1,84 @@
+name: Full CI with SonarQube
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)